### PR TITLE
added links to youtube videos in HTML5 module

### PR DESCRIPTION
**Pull request by nazmariam**
Hi, @OleksiyRudenko,  please check [HTML5 Forms module](https://github.com/kottans/frontend/blob/master/tasks/html5-forms.md) as an example of resolved issue "Add links to related youtube casting #276"

### CODE REVIEW

**Comment by OleksiyRudenko** [comment]
@nazmariam thanks

So, this example features a default thumbnail. Example:

1st default thumbnail
1st default thumbnail

Possible alternatives are

default resolution
default resolution thumbnail

medium resolution
medium resolution thumbnail

standard resolution
standard resolution thumbnail

I would stick to medium resolution thumbnail to save scrolling efforts.

Need more opinions.

@AMashoshyna @suchov @zonzujiro what do you think?

**Comment by OleksiyRudenko** [comment]
Yet another option (for the cases where a module relates to many videos) is to use a table.

E.g.

|   |    |
| --- | --- |
| default resolution thumbnail | [First workshop...](...)  |
| default resolution thumbnail | [Second workshop...](...)  |

**Comment by nazmariam** [comment]
@OleksiyRudenko OK, so you think it's good to use medium resolution images in single video modules and tables with small-resolution images in other cases. Let's wait for other opinions and I'll fix it

**Comment by yevhenorlov** [comment]
I agree with "use medium resolution images in single video modules and tables with small-resolution images in other cases"

**Comment by zonzujiro** [comment]
This is sad, that we don't have thumbnails for videos :(

**Comment by nazmariam** [comment]
@OleksiyRudenko I've renewed PR, please check next modules:
- [HTML5](https://github.com/nazmariam/frontend/blob/youtube-links/tasks/html5-forms.md#video)
- [WeatherApp](https://github.com/nazmariam/frontend/blob/youtube-links/tasks/weather-app.md#video)
- [ReactJS](https://github.com/nazmariam/frontend/blob/youtube-links/tasks/react.md#video)
- [Redux](https://github.com/nazmariam/frontend/blob/youtube-links/tasks/react-redux.md#video)
- [Git Advanced](https://github.com/nazmariam/frontend/blob/youtube-links/tasks/git-advanced.md#video)

File: tasks/git-advanced.md [hunk 1]
```diff
@@ -145,6 +145,14 @@ are rather a collection of possible theses.
 
 </details>
 
+## Video
+
+You can see workshop's screencast below.
```
Comment (OleksiyRudenko) [inline tasks/git-advanced.md]:
```suggestion
You can watch related screencasts from Kottans course live workshops below.
```

File: tasks/html5-forms.md [hunk 2]
```diff
@@ -59,6 +59,18 @@ Note: The list of recommended resources is extensive. You may not have enough ti
 - [Inhuman UI by Vadim Makeev](https://youtu.be/KAK-WAb9vow) - talk on accessibility and what web developers can do about it
 - [Weblind](https://weblind.ru/) - Recommendations on site development for people visually impaired
 
+#### Video
+
+You can see workshop's screencast below.
```
Comment (OleksiyRudenko) [inline tasks/html5-forms.md]:
```suggestion
You can watch related screencasts from Kottans course live workshops below.
```

File: tasks/react-redux.md [hunk 3]
```diff
@@ -19,6 +19,20 @@ We had a lot of reading resources about React & etc.
 But if you don't sure at component lifecycle yet,
 try to read this [article](https://habr.com/post/358090/).
 
+
+### Video
+
+You can see workshop's screencast below.
```
Comment (OleksiyRudenko) [inline tasks/react-redux.md]:
```suggestion
You can watch related screencasts from Kottans course live workshops below.
```

File: tasks/react.md [hunk 4]
```diff
@@ -31,7 +31,17 @@ When complete do the following:
    and share it with others –
    post a message in the [course channel][chat]:
    `React Friends App — #done` (or `React Friends App — #p2p_done` if you are p2p course student) and add the link to your repo. **This step is important, as it helps mentors to track your progress!**
-   
+
+#### Video
+
+You can see workshop's screencast below.
```
Comment (OleksiyRudenko) [inline tasks/react.md]:
```suggestion
You can watch related screencasts from Kottans course live workshops below.
```

**Comment by OleksiyRudenko** [comment]
Partially resolves #276

**Pull request merged**